etc/redfish-supermicro.dev

```diff
@@ -27,11 +29,7 @@
 # - A longer term solution is being investigated, see:
```

Comment:
Did you want to replace this comment with something like
> This device specification has been replaced with redfishpower-supermicro.dev, which does not require fragile delays to ensure power operations are complete before the script returns.  Users should migrate to the new one.

Reply:
did you maybe look at the wrong commit version?  I updated in https://github.com/chaos/powerman/pull/55/commits/835e52a15aee4f8c11e17810593b523af9e22bd8

Reply:
Oh I did have a stale version, sorry!